### ISSUE TITLE
Action menu layout broken after GM_registerMenuCommand is executed

### ISSUE THREAD

**Issue by TonyStark5** [Tampermonkey/tampermonkey#801]
(Please fill out the issue template with your details)

## Specifications

- Chromium: (i.e. 64.0.3282.140)
- TM: (i.e. 4.5.5648)
- OS: (i.e. MacOS 10.13.4)

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant none
// ==/UserScript==
```
点击油猴图标后，所有选项都在当前行重复出现，还很卡

**Comment by derjanb** [comment]
Can you please make a screenshot of the issue?

**Comment by SplitGemini** [comment]
I have this problem too.

**Comment by SplitGemini** [comment]
This problem occasionally occurs.
before:
![批注 2019-12-03 230059](https://user-images.githubusercontent.com/34861918/70062773-a1a68c80-1621-11ea-88d4-cb04191b5e4c.png)


problem appear:

![批注 2019-12-03 230425](https://user-images.githubusercontent.com/34861918/70062693-7f147380-1621-11ea-9784-5a049642d750.png)

I found scripts loaded more times:
![批注 2019-12-03 230442](https://user-images.githubusercontent.com/34861918/70062861-d4508500-1621-11ea-9049-6badb81958f9.png)

it can occur on any website, but it is not occur every times.

**Comment by SplitGemini** [comment]
My tampermonkey version is 4.9

**Comment by SplitGemini** [comment]
Chrome 78.0.3904.108 x64 , win10 1909
Have plugins: uBlock origin, Chrono, SwichyOmega, Stylus, OneTab, etc.

**Comment by SplitGemini** [comment]
And an important clue: click repeated label, it popup more than one pages.

**Comment by derjanb** [comment]
Should be fixed at 4.10.6104
